src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -348,7 +357,12 @@
                         this.updateComparisonPlotSelections)
                 }
             },
-            prepareOutputDownloads: mapActionByName("downloadResults", "prepareOutputs")
+            prepareOutputDownloads: mapActionByName("downloadResults", "prepareOutputs"),
+            handleClick(event: Event) {
```

Comment:
oh whoops that was a previous approach, i discovered something very weird about vue and html, will remove this but if interested read on!

Basically my initial approach was just a div overlay with `pointer-events: none` and a high `z-index` however did you know that point events actually will go through a high `z-index` element even if it has `pointer-events: none` on it and trigger the event in the element behind it? So I had this function capture and stop propagation of the click event because you could operate the map controls **through** the div overlay event though its pointer events were disabled. What is even weird is that once I made this a separate vue component (loading-tab) all of that resolved itself? I didn't need this function anymore. I think vue must handle events slightly differently underneath the hood if it has a nested vue component. Maybe it needs to hijack the DOM events and coordinate its own event? Not sure but yh thats why this was there. I found it very weird!